pkg/cmdlets/auth/signup.go

```diff
@@ -156,22 +161,23 @@ func promptForSignup(input *signupInput, out output.Outputer, prompter prompt.Pr
 		}
 	}
 
-	// Must define password validator here as it has to reference the input
-	var passwordValidator = func(val interface{}) error {
-		value := val.(string)
-		if value != input.Password {
-			return locale.NewError("InvalidPassword")
+	for i := 0; i < matchTries; i++ {
+		confirmMsg := locale.T("password_prompt_confirm")
+		input.Password2, err = prompter.InputSecret("", confirmMsg, prompt.InputRequired)
+		if err != nil {
+			return err
 		}
-		return nil
-	}
 
-	input.Password2, err = prompter.InputSecret("", locale.T("password_prompt_confirm"), prompt.InputRequired)
-	if err != nil {
-		return err
-	}
-	err = passwordValidator(input.Password2)
-	if err != nil {
-		return errs.Wrap(err, "InvalidPassword failure")
+		if input.Password2 == input.Password {
+			break
+		}
+
+		locErr := locale.NewError("err_password_confirmation_failed")
+		if i < matchTries-1 {
+			out.Notice(locErr.UserError())
```

Comment:
```suggestion
		if i < matchTries-1 {
			out.Notice(locale.T("err_password_confirmation_failed"))
```

Reply:
Yeah, I had wondered if you'd be OK with usage of UserError. It feels right to avoid localizing twice, but it's not common in our code. I opted to store the localization id as a var to use in both places.